mipengine/node/monetdb_interface/views.py

```diff
@@ -12,16 +12,32 @@ def get_view_names(context_id: str) -> List[str]:
 
 
 @validate_identifier_names
-def create_view(
+def create_pathology_view(
```

Comment:
we must try not "polluting" the db interface with concepts specific to the node. Does the monetdb_interface really need to "know" about the concept of pathologies? Why not have only the create_view function here?